### PR TITLE
Updates Structuring & Testing data success criteria

### PR DESCRIPTION
**Pull request by sampennington**
### Common Content?

- [ ] Block/s

### Common Theme?

- [ ] Yes

Issue number: #issue-number

### Org Content?

Module | Sprint | Page Type | Block Type

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: org-cyf-itp/content/structuring-data/success/index.md [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+
 +++
 
-Every module, you must review your progress to understand if your cohort can progress to the next stage. The conditions for success are listed below. If your cohort has met all of these conditions, you can progress to the next module. If you have not met them yet, what actions will you take to meet them?
+### 🎯 You've achieved your learning objectives if you can:
+
+```objectives
+- Define variables and functions.
+- Use if statements to write conditional code.
+- Understand the concepts of scope, return values and reusable functions.
```
Comment (illicitonion) [inline org-cyf-itp/content/structuring-data/success/index.md:21]: "Understand" is generally an avoided LO word (see https://docs.google.com/document/d/1NjeGQHCUFTplbj4tCA_lb-I9nugKRVh7PhcFwKXd-uo/edit) - can we expand this maybe to something like:

Identify the benefits of extracting a reusable function.
Explain what the result of calling a function evaluates to.
Explain which scopes can and cannot access variables declared in a JavaScript file.

(I'm not attached to these particular LOs, just some example ideas for how to make these more concrete)

File: org-cyf-itp/content/structuring-data/success/index.md [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
 weight = 11
 backlog= 'Module-JS1'
 [[objectives]]
-1="Distinguish interface from implementation"
-2="Translate requirements into unit tests"
-3="Interpret test feedback"
-4="Describe the intended behaviour of a program and its actual behaviour"
-5="Identify bugs in someone else’s code (person/machine)"
+1="Completed the [Intro to Python course](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/21)"
+2="Link to your exercises for the [Coursework Excercises](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/6) "
```
Comment (SallyMcGrath) [inline org-cyf-itp/content/structuring-data/success/index.md:11]:
```suggestion
2="Link to your exercises for the [Coursework Exercises](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/6) "
```

File: org-cyf-itp/content/structuring-data/success/index.md [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
 menu_level = ['module']
 weight = 11
 backlog= 'Module-JS1'
 [[objectives]]
-1="Distinguish interface from implementation"
-2="Translate requirements into unit tests"
-3="Interpret test feedback"
-4="Describe the intended behaviour of a program and its actual behaviour"
-5="Identify bugs in someone else’s code (person/machine)"
+1="Completed the [Intro to Python course](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/21)"
```
Comment (SallyMcGrath) [inline org-cyf-itp/content/structuring-data/success/index.md:10]:
```suggestion
1="Complete the [Intro to Python course](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/21)"
```

File: org-cyf-itp/content/structuring-data/success/index.md [hunk 4]
```diff
@@ -1,19 +1,35 @@
 +++
 title = 'success'
-description = 'How do we know if we have completed this module?'
+description = 'How do we know if we have completed the Structuring and Testing Data module?'
 layout = 'success'
 emoji= '✅'
 menu_level = ['module']
 weight = 11
 backlog= 'Module-JS1'
```
Comment (SallyMcGrath) [inline org-cyf-itp/content/structuring-data/success/index.md:8]:
```suggestion

```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
psst some typos

File: org-cyf-itp/content/structuring-data/success/index.md (hunk 1 above)
Comment (sampennington) [reply org-cyf-itp/content/structuring-data/success/index.md]: They sound pretty good! Have made some small changes, what do you think?

**Pull request merged**